docs/clients/99_graphql/index.rst

```diff
@@ -36,6 +36,20 @@ Pointing your browser to ``http://127.0.0.1:8888/explore``
 will bring up a `GraphiQL`_ interface to EdgeDB. This interface can be
 used to try out queries and explore the GraphQL capabilities.
 
+
+Known Limitations
+=================
+
+- Due to the differences between EdgeQL and GraphQL syntax
+  :eql:type:`enum <std::enum>` types which have values that use
+  symbols not found in identifiers (e.g. ``'N/A'`` or ``'NOT
```

Comment:
"types which have values that use symbols not found in identifiers" -> 'enum types that have values that cannot be represented as GraphQL identifiers (e.g. "N/A")'

Reply:
Fair enough. I'll update.